ocf_data_sampler/load/site.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+import xarray as xr
+
+from ocf_data_sampler.config.model import Sites
+
+
+def open_site(sites_config: Sites) -> xr.DataArray:
+
+    # Load site generation xr.Dataset
+    data_ds = xr.open_dataset(sites_config.filename)
+
+    # Load site generation data
+    metadata_df = pd.read_csv(sites_config.metadata_filename)
+
+    # LEGACY SUPPORT
+    data_ds = legacy_format(data_ds, metadata_df)
+
+    metadata_df.set_index("site_id", inplace=True, drop=True)
+
+    # Add coordinates
+    ds = data_ds.assign_coords(
+        latitude=(metadata_df.latitude.to_xarray()),
+        longitude=(metadata_df.longitude.to_xarray()),
+        capacity_kwp=data_ds.capacity_kwp,
+    )
+
+    return ds.generation_kw
+
+
+def legacy_format(data_ds, metadata_df):
```

Comment:
Here's a really big question, do we include stuff like this? 
It was useful for testing, so that I can see batched could be made, but in the future it would be good to remove it

Reply:
I want to remove this, and put it in a script for people to use, if they want